### ISSUE TITLE
Could not find org.jooq:jooq:3.11.2

### ISSUE THREAD

**Issue by batousik** [etiennestuder/gradle-jooq-plugin#86]
Hi!

I dont have internet access from my dev machine, so I have installed jooq 3.11.5 + your plugin to company maven repo. Can't build project since jooq-plugin requires dependency jooq 3.11.2

Regards,
Bato

**Comment by batousik** [comment]
Specified classpath dependencies in build script for all jooq libs to with version 3.11.5 - please fix that dependency requirement for people without internet connection

**Comment by etiennestuder** [comment]
Not sure what you are asking for - you can define the jooq version to apply during dependency resolution: https://github.com/etiennestuder/gradle-jooq-plugin#specifying-the-jooq-version-and-edition

**Comment by batousik** [comment]
No, this dep resolution happens before, in buildscript. I don't have internet connection and the only version installed in repo is 3.11.5. And you have a hardcoded version 3.11.2 in your plugin dependencies

**Comment by shellwalker** [comment]
I have the same problem when I using the springboot framework.

You can try to fix it by adding the following configuration into you `build.gradle` file.
`
dependencyManagement {
    // io.spring.dependency-management would break some dependencies
    dependencies {
        dependency ('org.jooq:jooq:3.11.5')
        dependency ('org.jooq:jooq-codegen:3.11.5')
        dependency ('org.jooq:jooq-meta:3.11.5')
    }
}
`

**Comment by etiennestuder** [comment]
@shellwalker Have you read this issue? https://github.com/etiennestuder/gradle-jooq-plugin/issues/81

**Comment by batousik** [comment]
Thank you for your replies.
I did add springframework - core as a dependency, and unfortunately i havent seen that bug. This explains everything i guess.
